userprofile/templates/userprofile/login.html

```diff
@@ -1,99 +1,115 @@
-            <div class="caption">
-                {% if shibboleth_title_text %}
-                <h3>
-                	{{ shibboleth_title_text|safe }}
-                </h3>
-                {% endif %}
-                {% if shibboleth_body_text %}
-                <p>
-                	{{ shibboleth_body_text|safe }}
-                </p>
-                {% endif %}
-                <p>
-                	<a class="btn btn-primary btn-block" role="button" href="{% url 'shibboleth-login' %}?next={{ next|iriencode }}">
-                		{{ shibboleth_button_text|safe }}
-                	</a>
-                </p>
-            </div>
-        </div>
-    </div>
-    {% endif %}
+		{% if mooc_login %}
+			<div class="col-sm-6 col-md-3 login-box">
+				<div class="thumbnail">
+					<div class="caption">
+						{% if mooc_title_text %}
+							<h3>
+								{{ mooc_title_text|safe }}
+							</h3>
+						{% endif %}
+						{% if mooc_body_text %}
+							<p>
+								{{ mooc_body_text|safe }}
+							</p>
+						{% endif %}
+						<p>
+							<a class="btn btn-danger btn-block" role="button" href="{% url 'social:begin' 'google-oauth2' %}?next={{ next|iriencode }}">{% trans 'Log in using Google' %}</a>
+						</p>
+					</div>
+				</div>
+			</div>
+		{% endif %}
 
-    {% if mooc_login %}
-    <div class="col-sm-6 col-md-3">
-        <div class="thumbnail">
-            <div class="caption">
-                {% if mooc_title_text %}
-                <h3>
-                  {{ mooc_title_text|safe }}
-                </h3>
-                {% endif %}
-                {% if mooc_body_text %}
-                <p>
-                    {{ mooc_body_text|safe }}
-                </p>
-                {% endif %}
-                <p>
-                    <a class="btn btn-danger btn-block" role="button" href="{% url 'social:begin' 'google-oauth2' %}?next={{ next|iriencode }}">{% trans 'Login Using Google' %}</a>
-                </p>
-            </div>
-        </div>
-    </div>
-    {% endif %}
+		<div class="col-sm-6 col-md-3 login-box extra-login">
+			<div class="thumbnail">
+				<div class="caption">
+					<h3>
+						{% trans "Local users" %}
+					</h3>
+					<p>
+						{% blocktrans trimmed %}
+							If you have been provided with credentials specifically
+							for {{ brand_name }}, use this login.
+						{% endblocktrans %}
+					</p>
+					<form method="post" role="form">
+						{% csrf_token %}
+						{#{ form|bootstrap }#}
+						<div class="form-group">
+							<input class="form-control" type="text" id="id_username" name="username" maxlength="254" placeholder="{% trans 'Username' %}">
+						</div>
+						<div class="form-group">
+							<input class="form-control" type="password" id="id_password" name="password" placeholder="{% trans 'Password' %}">
+						</div>
+						<div class="form-group">
+							<button type="submit" class="btn btn-default btn-block">{% trans "Log in" %}</button>
+						</div>
+					</form>
```

Comment:
I did the "Matthew-accessibility-check" and seems like it says there are few errors here, mainly the form-labels are missing: 

![image](https://user-images.githubusercontent.com/43036333/84796419-465a4b80-b001-11ea-9f1f-c4142fdae759.png)

Reply:
We talked that we leave this for the later. I finally created an issue about that #582. Thanks for testing the accessibility!